### PR TITLE
Changing font to Lato

### PR DESCRIPTION
**Pull request by dapiechnik**
for scroll link text. A really important change. I probably did it wrong. It felt like it was pulling from the _constants but I couldn't figure out how to change it.

### CODE REVIEW

File: source/assets/styles/pages/index/_scroll_link.scss [hunk 1]
```diff
@@ -1,6 +1,8 @@
 .scroll-link {
   @include heading;
   display: block;
+  font-family: $font-family-body;
```
Comment (Sinetheta) [inline source/assets/styles/pages/index/_scroll_link.scss:4]: It's the `@include heading;` above that changed the `.scroll_link`s font-family, so we'd probably start by getting rid of that, if we want regular body font.

**Comment by adamish717** [comment]
@dapiechnik Do you want to change this, or should I?

**Comment by dapiechnik** [comment]
I thought I did. What happened to my change? You can if you like.

**Comment by adamish717** [comment]
@dapiechnik Sorry, this got confusing and old and I broke it...so I removed the branch and just made the change on Master

**Pull request closed without merging**